### PR TITLE
Update create boundary conditions along polylines

### PR DESCRIPTION
**Pull request by TomFischer**
The tool is now able to write boundary conditions at points along polylines for process types liquid flow and groundwater flow and their primary variables PRESSURE1 and HEAD, respectively. The default value for PRESSSURE1 is zero, for HEAD is the z coordinate of the point the boundary condition is set on.

### CODE REVIEW

File: Applications/Utils/MeshEdit/CreateBoundaryConditionsAlongPolylines.cpp [hunk 1]
```diff
@@ -65,30 +65,61 @@ void convertMeshNodesToGeometry(std::vector<MeshLib::Node*> const& nodes,
...
+	bc_out << "#BOUNDARY_CONDITION\n";
+	bc_out << "  $PCS_TYPE\n";
+	bc_out << "    GROUNDWATER_FLOW\n";
+	bc_out << "  $PRIMARY_VARIABLE\n";
+	bc_out << "    HEAD\n";
+	bc_out << "  $GEO_TYPE\n";
+	bc_out << "    POINT " << pnt_name << "\n";
+	bc_out << "  $DIS_TYPE\n";
+	bc_out << "    CONSTANT " << head_value << "\n";
+}
```
Comment (endJunction) [inline Applications/Utils/MeshEdit/CreateBoundaryConditionsAlongPolylines.cpp]: For next PR: The output should be collected in one place. Currently there are three places (files) with similar outputs.

**Comment by endJunction** [comment]
:fast_forward:

File: Applications/Utils/MeshEdit/CreateBoundaryConditionsAlongPolylines.cpp (hunk 1 above)
Comment (TomFischer) [inline Applications/Utils/MeshEdit/CreateBoundaryConditionsAlongPolylines.cpp]: Checked the two other occurrences. The output was in two very old unused, unmaintained tools from 2011. I removed these tools.

**Comment by bilke** [comment]
:+1:

**Comment by TomFischer** [comment]
Jenkins, test this please.

**Comment by candysoul** [comment]
Successfully tested in Naegelstedt basin model. Thanks!

**Comment by UFZ-MJ** [comment]
Successfully tested in Naegelstedt catchment model. Thanks!

**Pull request merged**